content/docs/00-release-notes.md

```diff
@@ -73,7 +73,15 @@ Spectro Cloud Palette 3.0.0 is released with [Native Edge](/clusters/edge/native
 * Open Policy Agent version 3.9.0
 * AWS EBS  CSI version 1.10.0
 
+**Known Issues**
+
+* While deploying multiple apps in a sandbox cluster, if the deployment of one of the apps is blocked due to errors then subsequent apps deployed to the same sandbox cluster might also be stuck in deploying state.
 
+   **Suggested Workaround:**
+  * Delete the stuck App.
+  * Fix the App with the error.
+  * Redeploy the App again.
+      
```

Comment:
I recommended we remove the second use of bold text and just tell the reader inline that three workarounds are available. 

```suggestion
**Known Issues**

* While deploying multiple apps in a sandbox cluster, if the deployment of one of the apps is blocked due to errors, then subsequent apps deployed to the same sandbox cluster might also be stuck in deploying state.  Apply the following workarounds if you encounter the issue.

  * Delete the stuck App.
  * Fix the App with the error.
  * Redeploy the App again.  
```